### PR TITLE
[Intermediate Representation] Enable Hive e2e test

### LINKED ISSUE #1059: [Intermediate Representation] Enable Hive e2e test
Enable Hive e2e test to test IR.

### PR DESCRIPTION
**Pull request by Yancey0623**
fixed #1059

### CODE REVIEW

File: cmd/sqlflowserver/main_test.go [hunk 1]
```diff
@@ -335,6 +335,36 @@ func TestEnd2EndHive(t *testing.T) {
...
+	waitPortReady(fmt.Sprintf("localhost:%d", unitestPort), 0)
+	err = prepareTestData(dbConnStr)
+	if err != nil {
+		t.Fatalf("prepare test dataset failed: %v", err)
+	}
+	t.Run("TestShowDatabases", CaseShowDatabases)
+	t.Run("TestSelect", CaseSelect)
+	t.Run("TestTrainSQL", CaseTrainSQL)
+	t.Run("CaseTrainCustomModel", CaseTrainCustomModel)
+	t.Run("CaseTrainDeepWideModel", CaseTrainDeepWideModel)
```
Comment (typhoonzero) [inline cmd/sqlflowserver/main_test.go:414]: Add xgboost case?

File: pkg/sql/codegen/tensorflow/codegen.go [hunk 2]
```diff
@@ -249,6 +251,10 @@ func Pred(ir *codegen.PredictIR) (string, error) {
 		Y:                 ir.TrainIR.Label.GetFieldMeta()[0],
 		ModelParams:       modelParams,
 		Save:              "model_save",
+		HDFSNameNodeAddr:  session.HdfsNamenodeAddr,
```
Comment (typhoonzero) [inline pkg/sql/codegen/tensorflow/codegen.go:254]: Seems no test can ensure that these fields are filled?

Comment (Yancey0623) [reply pkg/sql/codegen/tensorflow/codegen.go:254]: Using `regex` to ensure the `HDFS...` field.

**Review by typhoonzero — APPROVED** [review]
LGTM++

**Pull request merged**